install/streams.csv

```diff
@@ -2429,7 +2449,8 @@ WIGC,20,B,N,50,true,false,false,2010-03-25T05:00:00Z,2020-06-16T00:00:00Z
 WIGC,20,H,N,200,true,false,true,2004-11-27T00:00:00Z,9999-01-01T00:00:00Z
 WILA,10,S,H,50,false,false,true,1994-06-21T21:27:00Z,1994-06-28T23:34:00Z
 WIZ,10,L,H,1,false,false,false,2007-04-19T04:00:00Z,9999-01-01T00:00:00Z
-WIZ,10,E,H,100,false,false,false,2006-09-19T00:00:01Z,9999-01-01T00:00:00Z
+WIZ,10,E,H,100,false,false,false,2006-09-19T00:00:01Z,2007-04-19T03:00:00Z
+WIZ,10,H,H,100,false,false,false,2007-04-19T04:00:00Z,9999-01-01T00:00:00Z
 WIZ,30,L,D,1,false,false,false,2006-09-19T00:00:01Z,9999-01-01T00:00:00Z
```

Comment:
This looks fine, but some comments on other existing metadata
- `sites.csv` & `sensors.csv` are aligned, noting that WIZ at LC 10 was opened in 1988.
- However, metadata at `streams.csv` only started in 2006.

Reply:
This will be related to the transition from analogue to digital, there's probably work to do here looking at historical connections and any early data (most likely triggered) which will not have good response information.